contributing/content-style-guide.md

```diff
@@ -340,6 +341,77 @@ A table following an H3 heading:
 > | `-readable` | Print output in human readable format. | <ul><li>`true`</li><li>`false`</li></ul> |
 > | `-file` | Print output to file instead of stdout. | Name of the file. |
 
+## Cards
+
+Use the `<Card` component to display content and actions on a single topic. Users should cards easy to scan for relevant and actionable information. Elements, like text, icons, and links can be used inside a card. 
+  
+You can configure a card in 2, 3, 4, or 5-column grids. To maintain a good user experience, be mindful of how the cards are displayed. Cards with multiple paragraphs may not work because the text and cards will appear too squished, making them hard to read and a bad user experience. 
+
+4 or 5-column grids should only be used:
+
+- Sparingly. There won't be many instances where you need to display 4 or 5 cards in a row on the docs site.
+- Only for cards that contain very little text, such as 15 words or less. 
```

Comment:
Leona - set it as a standard to keep it as 15 words or less. Link to examples
Matt -  Clarify why they want to use sparingly and that users prob want to use 2-3 columns. or remove sparingly bullet.

Reply:
i'm thinking about the use cases for Cards (eg how would a new contributor understand when to use to this component). i know we use them when creating landing pages. is there anything else? 

for their thoughts, add-ing @runleonarun and @matthewshaver to this convo

Reply:
oh yea, ok so def for the supported data platforms page but that won't be often (most likely will be a townie). i can imagine most cards usage would be something we (docs team or dx) would use since landing pages require more documentarian-type work and research - wdyt?

Reply:
oh yah, the supported data platforms page looks like another use case. what should that use case be called? i don't think it's a landing page (where there are links to its subpages)

Reply:
oh i'm not clear on this or what you mean, maybe we can chat about this in the 1:1 so we can get this finalized. maybe we need to clarify what exactly a landing page is as well.

the threads here have gotten a bit long and maybe I'm mixing different pr's I'm working on so perhaps its best to chat about this during the 1:1 😅